frontend/src/atoms/DatePickers.tsx

```diff
@@ -28,13 +28,16 @@ export const DatePickers:VFC=memo(()=> {
 
     const createNewDate=(props: string | number | Date ) =>{
```

Comment:
```suggestion
    const createNewDate = (props: string | number | Date ) => {
```

formatterを入れて整形するようにしたほうがいいですね
このアプリケーションではやらなくてもいいですが、次のアプリケーションでは必須ですね
(prettierを入れるといいです)

Reply:
あと、型定義は適切ですか？
Dateもstringも取るようにする必要あるんでしょうか
通常どちらかだと思いますが

Reply:
また、関数なのにpropsという命名の引数を受け取るのはおかしいので、適切な命名をしてください
(dateとかでいいです)